frontend/javascripts/oxalis/view/right-menu/dataset_info_tab_view.js

```diff
@@ -291,7 +288,7 @@ class DatasetInfoTabView extends React.PureComponent<Props> {
     }
 
     return (
-      <div className="flex-overflow">
+      <div>
```

Comment:
Because of `flex-overflow` the div had overflow y enabled. This lead to an always visible scrolling bar. As this annoyed me, I removed it.